### ISSUE TITLE
Modern Robotics Range sensor with Rev hub causes the program to hang

### ISSUE THREAD

**Issue by FestiveInvader** [ftctechnh/ftc_app#400]
I was trying to use the Modern Robotics Range sensor with the Rev module, and the program stops whenever I try to use the range sensor.  Replacing the MR range with a Rev Color/Distance fixed the problem, but I'd prefer to use the longer range Range.

**Comment by qwertychouskie** [comment]
For the sensors, you have to use an additional adapter wire along with the level converter.  Do you have this plugged in?

**Comment by FestiveInvader** [comment]
Yes, I have the crossover cable along with the logic level converter plugged in, and have tried flipping different connections around in case one was wrong. 

Note that not only was my code not working, but also the sample opmode. 

Thanks,
-Eric

**Comment by magneticflux-** [comment]
@FestiveInvader I'm not certain, but flipping the connectors around may have damaged the equipment. It probably *shouldn't*, but I don't know the exact specs of the sensor and I've heard of it happening to other things.

**Comment by FestiveInvader** [comment]
We have multiple sensors, as well as multiple Rev modules. I did refer to the wiring diagram to get the proper wiring the first time, but after that didn't work I flipped it. I have tried all three range sensors we have and none of them work. I haven't tried to use them with the Modern Robotics CDIM again, but as of last testing (mid July) the Range still worked. That was on v3.2 if I recall correctly. 

Thanks,
-Eric
